packages/ceramic-http-client/src/ceramic-http-client.ts

```diff
@@ -38,10 +37,10 @@ class CeramicClient implements CeramicApi {
         const result = await fetchJson(url)
         const { pinnedDocIds } = result
         return {
-          [Symbol.asyncIterator]() {
+          [Symbol.asyncIterator](): any {
             let index = 0
             return {
-              next() {
+              next(): any {
```

Comment:
Should be `Promise<IteratorResult<string>>` I think

Reply:
👍